samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-create-and-use-named-range-item
+name: Create and use named range item
+description: Create and use a named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+
+                sheet.names.add("ExpensesHeader", headerRange);
+
+                const namedItems = sheet.names.load();
+
+                await context.sync();
+
+                console.log("This workbook contains " + namedItems.items.length + " named items.");
+
+                for (let i = 0; i < namedItems.items.length; i++) {
+                    console.log(JSON.stringify(namedItems.items[i])) + "\n";
+                }   
+
+                await context.sync();
+            });
+        }
+
+        async function formatRangeByName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+
+                const headerRowName = sheet.names.getItem("ExpensesHeader");
+                const headerRange = headerRowName.getRange();
+                headerRange.format.fill.color = "red";
+
+                await context.sync();
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+
+                let sheet = await OfficeHelpers.ExcelUtilities.forceCreateSheet(context.workbook, "Sample");
+
+                let expensesTable = sheet.tables.add("A1:D1", true);
+                expensesTable.name = "ExpensesTable";
+
+                expensesTable.getHeaderRowRange().values = [["Date", "Merchant", "Category", "Amount"]];
+
+                let newData = transactions.map(item =>
+                    [item.date, item.merchant, item.category, item.amount]);
+
+                expensesTable.rows.add(null, newData);
```

Comment:
could you inline a comment for `null`, specifying what that parameter was?

    expensesTable.rows.add(null /*paramName*/, newData);